src/app/cube/details/details.component.ts

```diff
@@ -278,9 +282,9 @@ export class DetailsComponent implements OnInit, OnDestroy {
     } catch (e) {
 
       /**
-      * TODO: change status to 404 when issue #149 is closed
-      * if server error is thrown, navigate to not-found page
-      */
+       * TODO: change status to 404 when issue #149 is closed
```

Comment:
Do we need TODO this comment? or can this be deleted?

Reply:
@pzalep1 This is the entire function. It looks like these 'todo' comments are implemented but it actually looks like this function needs some rework after emily's status updates PR.
![image](https://user-images.githubusercontent.com/55514493/150822318-878195ba-2a5e-40b0-8243-5e03dfce0e95.png)